### DISCUSSION TITLE
Gas

### DISCUSSION THREAD

**Discussion by Marklord-dev** [Cyfrin/foundry-full-course-cu#1952] [Q&A]
How is the gas for each transaction on a blockchain determined?

**Comment by EngrPips** [comment]
Hello @Marklord-dev, this `https://updraft.cyfrin.io/courses/blockchain-basics/basics/introduction-to-gas` and this `https://updraft.cyfrin.io/courses/blockchain-basics/basics/gas-in-depth` will satisfy your curiosity.
